src/bindings/python/tests/runtime.py

```diff
@@ -118,6 +118,7 @@ def __call__(self, *input_values: NumericData) -> List[NumericData]:
         if self.network_cache.get(str(input_shapes)) is None:
             function = self.function
             if self.function.is_dynamic():
+                function = function.clone()
```

Comment:
Is this `clone` applies only in case of dynamic inputs?

Reply:
Yes, we have discussed it a while ago and the problem was only for dynamic case, so there is no need to add additional function cloning for non dynamic case